pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -269,6 +270,7 @@ func addOauthContainerToDeployment(clientset kubernetes.Interface, deployment *a
 		}
 		updatedPodSpec := deployment.Spec.Template.Spec.DeepCopy()
 		//	updatedPodSpec := podSpec.DeepCopy()
+		updatedPodSpec.AutomountServiceAccountToken = proto.Bool(true)
```

Comment:
same here -- add comment explaining that it is intentially different from upstream